java/libphonenumber/src/com/google/i18n/phonenumbers/nano/Phonemetadata.java

```diff
@@ -207,6 +207,12 @@ public static NumberFormat parseFrom(
     // optional string possible_number_pattern = 3;
     public java.lang.String possibleNumberPattern;
 
+    // repeated int32 possible_length = 9;
```

Comment:
I tried to, but they sadly didn't quite work: they added some extra protoc comments that I removed, put it in the wrong dir, and didn't have "nano" in the output path; so I did some manual editing based on the currently-checked in file and moved it into the right dir. I tried first playing with the command but the docs are hopeless so gave up.